.github/workflows/push.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+on: [push]
+jobs:
+  build:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: git checkout
+      - uses: actions/cache@v2
+        id: yarn-cache-2 # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: /home/runner/work/xen-orchestra/**/node_modules
```

Comment:
Is this path a guarantee?
Shouldn't we use a environment variable or a relative path like on the doc?

https://github.com/actions/cache